swarm/network/stream/peer.go

```diff
@@ -177,11 +186,6 @@ func (p *Peer) SendPriority(ctx context.Context, msg interface{}, priority uint8
 
 // SendOfferedHashes sends OfferedHashesMsg protocol msg
 func (p *Peer) SendOfferedHashes(s *server, f, t uint64) error {
-	var sp opentracing.Span
```

Comment:
Why are you removing this span? I know it is not very useful right now, as it is not to the other messages, but I don't think we should remove it.

Reply:
I put it back